assets/templates/partials/filter.tmpl

```diff
@@ -13,10 +13,10 @@
         <input type="hidden" name="sort" value="{{$sort}}"/>
         <fieldset class="ons-fieldset">
             {{ range $index, $topicFilter := $topicFilters }}
-                <div class="ons-checkboxes__items">
+                <div class="ons-checkboxes__items" role="tree">
                     <span class="ons-checkboxes__item ons-checkboxes__item--no-border">
                         <span class="ons-checkbox ons-checkbox--no-border topic-filter">
-                            <input type="checkbox" id="topic-group-{{ $index }}"
+                            <input type="checkbox" role="group" id="topic-group-{{ $index }}"
```

Comment:
I'm not sure this group role is allowed on an input - I can't get at an Axe extension but I've run it through an axe crawler and w3c html checker and they both complain about it.

Reply:
Hmmm. Mine didn't complain when I ran. Will look again.